### PR TITLE
Add list-keys and generate-recovery-key command to CLI.

### PR DESCRIPTION
**Pull request by eolszewski**

### CODE REVIEW

File: packages/transmute-cli/src/commands/generate-recovery-key/gpg_import [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+: ${SECRETS_DIR:=$HOME/.transmute/cli-secrets}
+
+if type "gpg2" > /dev/null 2 > /dev/null; then
+  echo 'gpg2 detected'
+  set + e
+  echo 'Input your passphrase to import your recovery key into your GPG keyring.'
+  gpg2 --import $SECRETS_DIR/recovery_sk.key
```
Comment (OR13) [inline packages/transmute-cli/src/commands/generate-recovery-key/gpg_import]: feels like these should be script arguments, parsed by vorpal, and passed to bash via shelljs.

File: packages/transmute-cli/src/commands/generate-recovery-key/index.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+const run = require('../runner');
+
+const shell = require('shelljs');
+
+const { writeFile, readFile, removeFile } = require('../../utils');
+
+module.exports.generateRecoveryKey = async (args) => {
+
+  let transmuteDir = path.join(require('os').homedir(), '.transmute/cli-secrets');
+  shell.env.RECOVERY_KEY_FINGERPRINT = args.fingerprint;
```
Comment (OR13) [inline packages/transmute-cli/src/commands/generate-recovery-key/index.js]: cool, did not know we can do this.

Comment (OR13) [reply packages/transmute-cli/src/commands/generate-recovery-key/index.js]: Still wonder if this is better to be a script argument instead of env var.

Comment (eolszewski) [reply packages/transmute-cli/src/commands/generate-recovery-key/index.js]: Updated.

File: packages/transmute-cli/src/commands/generate-recovery-key/gpg_import (hunk 1 above)
Comment (eolszewski) [reply packages/transmute-cli/src/commands/generate-recovery-key/gpg_import]: Updated.

**Pull request merged**